styleguide/derek/view-templates/stories/stories.ejs

```diff
@@ -107,10 +110,14 @@ var featQuotes = [
         </div>
         <% } %>
         <div class="row">
-          <ul class="pager">
-            <li class="pager-previous first"><a href="javascript:void(0);">prev</a></li>
+          <ul class="pager rsFilter-storiesPager">
```

Comment:
@runrog this class `rsFilter-storiesPager` should be in a wrapper div. We don't want to add custom classes to the pager itself.